dns_server/Cargo.toml

```diff
@@ -27,3 +28,7 @@ parity-scale-codec.workspace = true
 void = "1.0"
 clap = { version = "4", features = ["derive"] }
 directories = "4.0"
+
+[dev-dependencies]
+rstest = "0.16"
+test-utils = { path = "../test-utils" }
```

Comment:
Nitpick: a new line between "foreign" and internal dependencies?.. 🙃

Reply:
I will fix, thanks.